locksmithctl/daemon.go

```diff
@@ -364,14 +380,7 @@ func runDaemon() int {
 	close(stop)
 	wg.Wait()
 
-	if period != nil {
-		now := time.Now()
-		sleeptime := period.DurationToStart(now)
-		if sleeptime > 0 {
-			dlog.Infof("Waiting for %s to reboot.", sleeptime)
-			time.Sleep(sleeptime)
-		}
-	}
+	getPeriodicAndSleep()
```

Comment:
same as above, we can handle the error:
```suggestion
	if err := getPeriodicAndSleep(); err != nil {
	     ...
```